packages/core/admin/admin/src/content-manager/pages/ListView/components/TableRows/index.js

```diff
@@ -178,19 +179,36 @@ export const TableRows = ({
                 );
               }
 
-              if (hasReviewWorkflows && name === 'strapi_reviewWorkflows_stage') {
-                return (
-                  <Td key={key}>
-                    {data.strapi_reviewWorkflows_stage ? (
-                      <ReviewWorkflowsStage
-                        color={data.strapi_reviewWorkflows_stage.color}
-                        name={data.strapi_reviewWorkflows_stage.name}
-                      />
-                    ) : (
-                      <Typography textColor="neutral800">-</Typography>
-                    )}
-                  </Td>
-                );
+              if (hasReviewWorkflows) {
+                if (name === 'strapi_reviewWorkflows_stage') {
+                  return (
+                    <Td key={key}>
+                      {data.strapi_reviewWorkflows_stage ? (
+                        <ReviewWorkflowsColumns.ReviewWorkflowsStageEE
+                          color={data.strapi_reviewWorkflows_stage.color}
+                          name={data.strapi_reviewWorkflows_stage.name}
+                        />
+                      ) : (
+                        <Typography textColor="neutral800">-</Typography>
+                      )}
+                    </Td>
+                  );
+                }
+                if (name === 'strapi_assignee') {
+                  return (
+                    <Td key={key}>
+                      {data.strapi_assignee ? (
+                        <ReviewWorkflowsColumns.ReviewWorkflowsAssigneeEE
+                          firstname={data.strapi_assignee.firstname}
+                          lastname={data?.strapi_assignee?.lastname ?? ''}
+                          displayname={data?.strapi_assignee?.displayname ?? ''}
```

Comment:
Why do you want the empty string fallback here? Why not just pass in `undefined`?